HogeFuga/library/src/main/java/com/example/library/HogeFuga.java

```diff
@@ -0,0 +1,26 @@
+package com.example.library;
+
+public class HogeFuga {
+    private HogeFuga hogefuga;
+
+    public void hogefuga() {
+        assert false;
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/WataruSuzuki/Android-Example-Using-Danger/blob/d44a3ff3897f5daace1e0ce43cb9b80e4928e0e5/HogeFuga/library/src/main/java/com/example/library/HogeFuga.java#L7"></span>Assertions are unreliable in Dalvik and unimplemented in ART. Use <code>BuildConfig.DEBUG</code> conditional checks instead.</td>
    </tr>
  </tbody>
</table>